src/scratchpad.js

```diff
@@ -132,7 +132,26 @@ export default function Scratchpad (DOM, props) {
     }
   };
 
+  const clientWidth$ = DOM.observable.pluck('clientWidth');
```

Comment:
Hiya, I'm actually getting an error on this line. I think instead we want `DOM.select(':root').observable.pluck('clientWidth');`

If you make that change I'm happy to merge 😄 👍